src/libgap-api.c

```diff
@@ -258,3 +258,16 @@ char * GAP_CSTR_STRING(Obj string)
         return 0;
     return CSTR_STRING(string);
 }
+
+UChar GAP_CharValue(Obj obj)
+{
+    if (TNUM_OBJ(obj) != T_CHAR) {
+        ErrorQuit("GAP_CharValue: argument is not a char", 0, 0);
+    }
+    return CHAR_VALUE(obj);
+}
```

Comment:
I don't know if some people might not want ErrorQuit calling. In that case you could return an Int, which is an unsigned char, or -1 if the value is not a char.